fadtk/model_loader.py

```diff
@@ -54,16 +56,21 @@ def _get_embedding(self, audio: np.ndarray):
     def load_wav(self, wav_file: Path):
         wav_data, _ = soundfile.read(wav_file, dtype='int16')
         wav_data = wav_data / 32768.0  # Convert to [-1.0, +1.0]
-
+        if self.audio_len is not None:
```

Comment:
can you add comments about what you intend to do here? it'd be very helpful for anyone reading this code in the future, no matter how straightforward this code is (because then they can clarify if your intention is aligned with the implementation or not.)